framework/src/modules/interoperability/base_cross_chain_update_command.ts

```diff
@@ -19,7 +19,7 @@ import { CommandExecuteContext, CommandVerifyContext } from '../../state_machine
 import { BaseInteroperabilityCommand } from './base_interoperability_command';
 import { BaseInteroperabilityInternalMethod } from './base_interoperability_internal_methods';
 import { BaseInteroperabilityMethod } from './base_interoperability_method';
-import { CCMStatusCode, EMPTY_BYTES, EmptyCCM } from './constants';
+import { CCMStatusCode, EMPTY_BYTES, EVENT_TOPIC_CCM_EXECUTION, EmptyCCM } from './constants';
```

Comment:
@Phanco i think change in this file is correct to use the 6.0 version since there were change in the LIP. Also, the main change from this [commit](https://github.com/LiskHQ/lisk-sdk/commit/d75cf3a7c731d587fbb24062b043b70ee41327ec) exists although some implementation seem some style is different